src/controller/plugins/Endpoints/TestEndpoints.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+
+namespace plugins.Endpoints;
+
+public static class TestEndpoints
+{
```

Comment:
I assume this is supposed to be here. Honestly let's just let it be...

Reply:
Hmmm, good question